examples/cart/index.js

```diff
@@ -201,4 +226,32 @@ $(function() {
     return '$' + parseFloat(amount, 10).toFixed(2).replace(/(\d)(?=(\d{3})+\.)/g, "$1,").toString();
   }
 
+
+  /* Checkout listener
+  ============================================================ */
+  function attachCheckoutButtonListeners() {
+    $('.btn--cart-checkout').on('click', function () {
+      window.open(cart.checkoutUrl, '_self');
+    });
+  }
+
+  /* Update cart tab button
+  ============================================================ */
+  function updateCartTabButton() {
+    if (cart.lineItems.length > 0) {
+      var totalItems = cart.lineItems.reduce(function(total, item) {
+        return total + item.quantity;
+      }, 0);
```

Comment:
Made a note to move this work to the cart: https://github.com/Shopify/js-buy-sdk/issues/51